app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -103,7 +108,7 @@ const AppLayoutWithExperiment = ({
     };
   }, [startPolling, stopPolling, experiment, polling]);
 
-  if (loading || (analysisRequired && !analysis)) {
+  if (loading || (analysisRequired && analysisStatus === "loading")) {
```

Comment:
FWIW the useAsync method does have a `loading` property. But I'm curious: One thing that is hard to test locally but I know @emtwo had done some work to try and avoid was flashing an error/unavailable state on the page if analysis results were still loading, and because the experiment request finishes loading at a separate time than the analysis request, I have a feeling this behavior may be re-introduced with this change. [This PR](https://github.com/mozilla/experimenter/pull/4233/files) has more details of those changes if you're curious.

Reply:
I noticed when testing this locally that (in addition to the issue with trying access the `undefined` object) if the server returns a `500` response and `analysis` is `undefined`,  `PageLoading` will show up forever – I'll take a look at the PR and see if I can reproduce though

Reply:
Oh yeah hm, the flashing does seem to be an issue. Lemme look again

Reply:
Yeah so my thinking with the 500 error issue causing indefinite loading is that there shouldn’t be a non-addressable scenario where that is occurring - outlined in more detail here: https://github.com/mozilla/experimenter/pull/4233#issuecomment-743397223

but I also recognize “it shouldn’t happen” doesn’t equal “we shouldn’t try to handle it”, so if there’s a better solution I’m super game

Reply:
Ok, I *think* the last commit I pushed works for this  by handling the loading and never-attempted-to-fetch

Reply:
I'm not seeing the updates for some reason? Did the push go through?